spring-batch-infrastructure/src/main/java/org/springframework/batch/item/file/mapping/DefaultLineMapper.java

```diff
@@ -38,7 +38,16 @@
 
 	private FieldSetMapper<T> fieldSetMapper;
 
-    @Override
+	public DefaultLineMapper(LineTokenizer tokenizer, FieldSetMapper<T> fieldSetMapper) {
+		setLineTokenizer(tokenizer);
+		setFieldSetMapper(fieldSetMapper);
+		afterPropertiesSet();
```

Comment:
I'm open to being convinced otherwise, but I doubt calling `IntializingBean#afterPropertiesSet()` from within the constructor of a class is considered a good practice...